dojo/models.py

```diff
@@ -832,11 +832,11 @@ def get_response(self):
 
 class Risk_Acceptance(models.Model):
     path = models.FileField(upload_to='risk/%Y/%m/%d',
-                            editable=False, null=False,
+                            editable=True, null=False,
```

Comment:
Hi @narayanaditya95  great catch. It looks like in 1.9 forward we use field.disabled

https://docs.djangoproject.com/en/1.9/ref/forms/fields/#disabled